pages/transactions/[id].js

```diff
@@ -358,123 +343,101 @@ const StyledSuccessMessage = styled.p`
   left: 50%;
   transform: translateX(-50%);
   color: var(--text-color-dark);
-  font-size: 0.8rem;
+  font-size: var(--font-size-xs);
+  line-height: 1.4;
   padding: 6px 4px;
   width: 220px;
   font-weight: 800;
   text-align: center;
   border-radius: 8px;
-  box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+  box-shadow: var(--shadow-brand);
   z-index: 1000;
   background-color: var(--friendly-green-color);
 `;
 
-const StyledBackLink = styled(Link)`
-  display: flex;
-  align-items: center;
-  background-color: var(--accent-color);
-  text-decoration: none;
-  width: fit-content;
-  color: black;
-  position: relative;
-  top: 10px;
-  left: 10px;
-  border: var(--accent-color);
-  padding: 4px 8px;
-  border-radius: 4px;
-  display: flex;
-  gap: 0.5rem;
-`;
-
 const StyledTransactionDetails = styled.div`
   display: flex;
   flex-direction: column;
   align-items: center;
-  gap: 8px;
+  gap: var(--gap-md);
 `;
 
 const StyledNoteArea = styled.div`
-  border: 0.1px solid var(--dark-grey-color);
   border-radius: 16px;
   padding: 8px 16px;
-  min-height: 14rem;
-  width: 18rem;
+  width: 300px;
   display: flex;
   flex-direction: column;
   background-color: var(--accent-color);
-  gap: 8px;
-`;
-
-const StyledTitle = styled.h2`
-  text-align: center;
-  font-size: 1.7rem;
-  font-weight: 700;
+  gap: var(--gap-xl);
 `;
 
 const StyledConfirmActionContainer = styled.div`
   display: flex;
   justify-content: center;
   align-items: center;
-  gap: 4px;
-  border: 0.1px solid var(--dark-grey-color);
+  gap: var(--gap-sm);
   border-radius: 16px;
   padding: 8px 16px;
-  width: 18rem;
-  min-height: 13rem;
-  background-color: white;
+  width: 300px;
+  border: ${(props) => (props.$isNote ? "none" : "var(--border-brand)")};
+  height: ${(props) => (props.$isNote ? "195px" : "250px")};
+  background-color: ${(props) =>
+    props.$isNote ? "var(--accent-color)" : "var(--white-bg-color)"};
```

Comment:
this can be combined so you only need one callback:
```suggestion
  border: var(--border-brand);
  height: 250px;
  background-color: var(--white-bg-color);
  
  ${(props) =>
    props.$isNote && css`
      border: none;
      height: 195px;
      background-color: var(--accent-color);
      `}
 
```